src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -206,12 +534,49 @@ public SDKRestClient restClient() {
         return this.sdkRestClient;
     }
 
-    // @Override
-    public Map<String, Class<? extends TransportAction<? extends ActionRequest, ? extends ActionResponse>>> getActionsMap() {
-        Map<String, Class<? extends TransportAction<? extends ActionRequest, ? extends ActionResponse>>> map = new HashMap<>();
-        map.put(ADJobParameterAction.NAME, ADJobParameterTransportAction.class);
-        map.put(ADJobRunnerAction.NAME, ADJobRunnerTransportAction.class);
-        return map;
+    /**
+     * createComponents doesn't work for Clock as ES process cannot start
```

Comment:
Also, if you can raise a PR on the `main` to correct this
```suggestion
     * createComponents doesn't work for Clock as OS process cannot start
```